lychee-lib/src/types/cache.rs

```diff
@@ -54,3 +76,61 @@ impl From<&Status> for CacheStatus {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use serde::de::value::{BorrowedStrDeserializer, Error as DeserializerError};
+    use serde::Deserialize;
+
+    use crate::CacheStatus;
+
+    #[test]
+    fn test_deserialize_cache_status_success_code() {
+        let deserializer: BorrowedStrDeserializer<DeserializerError> =
```

Comment:
Looks like this deserializer gets initiated for every test. How about creating a helper function for it?
```rust
fn deserialize_cache_status(s: &str) -> CacheStatus {
    let deserializer: BorrowedStrDeserializer<DeserializerError> =
        BorrowedStrDeserializer::new(s);
    CacheStatus::deserialize(deserializer)
}

assert_eq!(
    deserialize_cache_status("200"),
    Ok(CacheStatus::Ok(200))
);
```

Reply:
Will do!